docs/inso-cli/cli-command-reference/inso-generate-config.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: article-detail
+title:  inso generate config
+category: "CLI Command Reference"
+category-url: inso-cli
+---
+
+Similar to the Kong Kubernetes and Declarative config plugins for Insomnia, this command can generate configuration from an API specification, using openapi-2-kong.
+
+### Command
+
+`inso generate config [identifier]`
+
+Execute the given specification, the user will be prompted to select a specification if one is not passed as an option. This can also be a file on your filesystem.
```

Comment:
Very strange, I'm not sure where the "execute the given spec" wording came from because there's nothing to execute here ([original docs](https://github.com/Kong/insomnia/tree/develop/packages/insomnia-inso#-inso-generate-config-identifier)). Is this redundant given the summary at the top of this page, plus the fact that `identifier` is defined below?